src/ui_ng/lib/src/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -62,6 +61,13 @@ export class ConfirmationDialogComponent {
         return false;
     }
 
+    get isReplicate(): boolean {
+        if (this.dialogTitle && (this.dialogTitle.includes('REPLICATION') || this.dialogTitle.includes('replication'))) {
```

Comment:
i18 text change will make this logic failed, use mode property and corresponding const vars